lib/util/i18n.js

```diff
@@ -45,6 +47,33 @@ export function getConfigLocales(configLanguages) {
   return locales
 }
 
+/**
+ * Loads in all localized strings from @opentripplanner packages.
+ * Package list is loaded from the package.json and each yml file then imported.
+ *
+ * All strings are then merged
+ */
+async function loadOtpUiLocaleData(matchedLocale) {
+  const packageJson = await import('../../package.json')
+  const otpUiMessages = await Promise.all(
+    Object.keys(packageJson.dependencies)
+      .filter((pkg) => pkg.includes('@opentripplanner'))
```

Comment:
`.startsWith`?

Reply:
Good call! Refactored in bfe701697e8cbceef1b004a198352f7504f4e6b0